src/utils/express.ts

```diff
@@ -9,20 +9,19 @@ import {
  * which takes an express.Request as input and returns an IResponse,
  * into an express controller.
  */
-export function toExpressHandler<T>(
-  handler: (req: express.Request) => Promise<IResponse<T>>
-): <P>(req: express.Request, res: express.Response, object?: P) => void {
-  return async (req, res, object) => {
-    try {
-      const response = await handler.call(object, req);
-      // tslint:disable-next-line:no-object-mutation
-      res.locals.detail = response.detail;
-      response.apply(res);
-    } catch (e) {
-      const response = ResponseErrorInternal(e);
-      // tslint:disable-next-line:no-object-mutation
-      res.locals.detail = response.detail;
-      response.apply(res);
-    }
+export function toExpressHandler<T, P>(
+  handler: (req: express.Request) => Promise<IResponse<T>>,
+  object?: P
+): (req: express.Request, res: express.Response) => void {
+  return (req, res) => {
```

Comment:
remove this curly brace may let you remove the `tslint:disable` directive on the next line

Reply:
Fixed in e4acea1